ext/opcache/ZendAccelerator.c

```diff
@@ -3692,9 +3633,8 @@ static zend_op_array *preload_compile_file(zend_file_handle *file_handle, int ty
 //???		efree(op_array->refcount);
 		op_array->refcount = NULL;
 
-		if (op_array->static_variables &&
-		    !(GC_FLAGS(op_array->static_variables) & IS_ARRAY_IMMUTABLE)) {
-			GC_ADDREF(op_array->static_variables);
+		if (op_array->static_variables) {
+			GC_TRY_ADDREF(op_array->static_variables);
```

Comment:
Why do we need this increment? I guess, we did this to trigger CoW and keep original values.

Reply:
Yes, this addref is not needed anymore. I've dropped it.